src/main/resources/META-INF/jpa-changelog-phasetwo-20240308.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+  <changeSet author="xgp" id="invite-url-len">
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <dbms type="cockroachdb"/>
+      </not>
+    </preConditions>
+    <modifyDataType tableName="INVITATION" columnName="URL" newDataType="varchar(2048)" />
+  </changeSet>
+
+  <!-- crdb specific -->
+  <changeSet author="xgp" id="invite-url-len-crdb">
+    <preConditions onFail="MARK_RAN">
+      <dbms type="cockroachdb"/>
+    </preConditions>
+    <sql dbms="cockroachdb">SET enable_experimental_alter_column_type_general=true;</sql>
```

Comment:
Just an observation. This "SET enable_experimental_alter_column_type_general=true;" must be set before each alter table operation. It's a limitation of cockroachdb right?

Reply:
Correct. And it only needs to be set once per "session". In this case, the whole changeset is run in one transaction, so if there were more than one query, as long as they are in the same transaction, it only needs to be run once.